product-service/src/handlers/getProducts.js

```diff
@@ -0,0 +1,7 @@
+import { getAllProducts } from '../services/getAllProducts/getAllProducts';
+import { buildHttpResponse } from '../utils/buildHttpResponse';
+
+export const getProducts = async () => {
```

Comment:
What if any error will be thrown? Will be a good idea to handle it with try catch block and return at least 500 error.